### PR TITLE
Fixed Grammer mistakes of control_flow_op.py

### PR DESCRIPTION
**Pull request by mohantym**
Fixed some Grammar mistakes suggested by Pycharm editor

### CODE REVIEW

File: tensorflow/python/autograph/operators/control_flow.py [hunk 1]
```diff
@@ -405,7 +405,7 @@ def for_stmt(iter_, extra_test, body, get_state, set_state, symbol_names, opts):
       arith_mean += a
   ```
 
-  The state is represented by the variables geo_mean and arith_mean. The
+  The state is represented by the variable's geo_mean and arith_mean. The
```
Comment (mihaimaruseac) [inline tensorflow/python/autograph/operators/control_flow.py:408]: This is wrong. There are 2 variables.

File: tensorflow/python/autograph/operators/control_flow.py [hunk 2]
```diff
@@ -1127,7 +1127,7 @@ def _try_handling_undefineds(body, get_state, set_state, init_vars, nulls,
    1. the types of loop variables could be inferred (usually by staging one
        iteration)
    2. these types could be replaced by placeholders (e.g. zero values, for
-       tensors.
+       tensors.)
```
Comment (mihaimaruseac) [inline tensorflow/python/autograph/operators/control_flow.py:1130]: `.` outside of `)`

File: tensorflow/python/autograph/operators/control_flow.py [hunk 3]
```diff
@@ -1344,7 +1344,7 @@ def if_stmt(cond, body, orelse, get_state, set_state, symbol_names, nouts):
       state that branches may mutate through side effects. This function is not
       needed and should not be called when dispatching to code matching Python's
       default semantics. This is useful for checkpointing to avoid unintended
-      side-effects when staging requires evaluating all code-paths.
+      side effects when staging requires evaluating all code-paths.
```
Comment (mihaimaruseac) [inline tensorflow/python/autograph/operators/control_flow.py:1347]: `-` here is good (either form works so lt's not change needlessly)

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

File: tensorflow/python/autograph/operators/control_flow.py (hunk 3 above)
Comment (mohantym) [reply tensorflow/python/autograph/operators/control_flow.py]: Thanks mihai for the review. updated as instructed.

File: tensorflow/python/autograph/operators/control_flow.py [hunk 4]
```diff
@@ -405,7 +405,7 @@ def for_stmt(iter_, extra_test, body, get_state, set_state, symbol_names, opts):
       arith_mean += a
   ```
 
-  The state is represented by the variables geo_mean and arith_mean. The
+  The state is represented by variables named geo_mean and arith_mean. The
```
Comment (mihaimaruseac) [inline tensorflow/python/autograph/operators/control_flow.py:408]: `the` is good here.

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

**Comment by gbaned** [comment]
@mohantym Can you please check @mihaimaruseac's comments and keep us posted ? Thank you!

File: tensorflow/python/autograph/operators/control_flow.py (hunk 4 above)
Comment (mihaimaruseac) [inline tensorflow/python/autograph/operators/control_flow.py:408]:
```suggestion
  The state is represented by the variables named geo_mean and arith_mean. The
```

**Review by mihaimaruseac — APPROVED** [review]
Fixing it myself :P

File: tensorflow/python/autograph/operators/control_flow.py (hunk 4 above)
Comment (mihaimaruseac) [reply tensorflow/python/autograph/operators/control_flow.py]: Fixing it myself

**Pull request merged**